drivers/charger/charger_bq25180.c

```diff
@@ -17,31 +17,34 @@
 
 LOG_MODULE_REGISTER(bq25180, CONFIG_CHARGER_LOG_LEVEL);
 
-#define BQ25180_STAT0 0x00
-#define BQ25180_STAT1 0x01
-#define BQ25180_FLAG0 0x02
+#define BQ25180_STAT0     0x00
+#define BQ25180_STAT1     0x01
+#define BQ25180_FLAG0     0x02
 #define BQ25180_VBAT_CTRL 0x03
 #define BQ25180_ICHG_CTRL 0x04
-#define BQ25180_IC_CTRL 0x07
-#define BQ25180_SHIP_RST 0x09
-#define BQ25180_MASK_ID 0x0c
+#define BQ25180_IC_CTRL   0x07
+#define BQ25180_SHIP_RST  0x09
+#define BQ25180_MASK_ID   0x0c
 
-#define BQ25180_STAT0_CHG_STAT_MASK GENMASK(6, 5)
-#define BQ25180_STAT0_CHG_STAT_NOT_CHARGING 0x00
+#define BQ25180_STAT0_CHG_STAT_MASK             GENMASK(6, 5)
+#define BQ25180_STAT0_CHG_STAT_NOT_CHARGING     0x00
 #define BQ25180_STAT0_CHG_STAT_CONSTANT_CURRENT 0x01
 #define BQ25180_STAT0_CHG_STAT_CONSTANT_VOLTAGE 0x02
-#define BQ25180_STAT0_CHG_STAT_DONE 0x03
-#define BQ25180_STAT0_VIN_PGOOD_STAT BIT(0)
+#define BQ25180_STAT0_CHG_STAT_DONE             0x03
+#define BQ25180_STAT0_VIN_PGOOD_STAT            BIT(0)
 #define BQ25180_VBAT_MSK                        GENMASK(6, 0)
-#define BQ25180_ICHG_CHG_DIS BIT(7)
-#define BQ25180_ICHG_MSK GENMASK(6, 0)
-#define BQ25180_WATCHDOG_SEL_1_MSK GENMASK(1, 0)
-#define BQ25180_WATCHDOG_DISABLE 0x03
-#define BQ25180_DEVICE_ID_MSK GENMASK(3, 0)
-#define BQ25180_DEVICE_ID 0x00
-#define BQ25180_SHIP_RST_EN_RST_SHIP_MSK GENMASK(6, 5)
-#define BQ25180_SHIP_RST_EN_RST_SHIP_ADAPTER 0x20
-#define BQ25180_SHIP_RST_EN_RST_SHIP_BUTTON 0x40
+#define BQ25180_ICHG_CHG_DIS                    BIT(7)
+#define BQ25180_ICHG_MSK                        GENMASK(6, 0)
+#define BQ25180_IC_CTRL_VRCH_100                0x00
+#define BQ25180_IC_CTRL_VRCH_200                BIT(5)
+#define BQ25180_IC_CTRL_VRCH_MSK                BIT(5)
+#define BQ25180_WATCHDOG_SEL_1_MSK              GENMASK(1, 0)
+#define BQ25180_WATCHDOG_DISABLE                0x03
+#define BQ25180_DEVICE_ID_MSK                   GENMASK(3, 0)
+#define BQ25180_DEVICE_ID                       0x00
+#define BQ25180_SHIP_RST_EN_RST_SHIP_MSK        GENMASK(6, 5)
+#define BQ25180_SHIP_RST_EN_RST_SHIP_ADAPTER    0x20
+#define BQ25180_SHIP_RST_EN_RST_SHIP_BUTTON     0x40
```

Comment:
``` c
#define BQ25180_IC_CTRL_VRCH_100                0x00
#define BQ25180_WATCHDOG_DISABLE 0x03	#define BQ25180_IC_CTRL_VRCH_200                BIT(5)
#define BQ25180_DEVICE_ID_MSK GENMASK(3, 0)	#define BQ25180_IC_CTRL_VRCH_MSK                BIT(5)
```
Only these lines should be introduced. Using `git clang-format` would only format your staged changes and avoid spurious edits.

Reply:
Changed :)